client_unix_test.go

```diff
@@ -2,12 +2,30 @@
 
 package containerd
 
+import (
+	"runtime"
+)
+
 const (
 	defaultRoot    = "/var/lib/containerd-test"
 	defaultAddress = "/run/containerd-test/containerd.sock"
-	testImage      = "docker.io/library/alpine:latest"
+)
+
+var (
+	testImage string
 )
 
 func platformTestSetup(client *Client) error {
 	return nil
 }
+
+func init() {
+	switch runtime.GOARCH {
+	case "amd64":
+		testImage = "docker.io/library/alpine:latest"
+	case "arm":
+		testImage = "docker.io/armhf/alpine:latest"
+	default:
+		testImage = "docker.io/" + runtime.GOARCH + "/alpine:latest"
```

Comment:
@estesp well to be fair i would expect multi-arch images to be manifest lists under the same namespace